data-catalog-entry-schema.json

```diff
@@ -0,0 +1,263 @@
+          "email": {
+            "title": "Email",
+            "anyOf": [
+              {
+                "type": "string",
+                "format": "email"
+              },
+              {
+                "type": "string",
+                "maxLength": 0
+              }
+            ]
+          },
+          "role": {
+            "title": "Role",
+            "description": "What is role in project (dropdown menu)",
+            "type": "string",
+            "enum": [
+              "PI",
+              "PhD Student",
+              "Masters Student",
+              "Collaborator",
+              "HIWI",
+              "Field Assistant",
+              "Lab Assistant",
+              "Analyst"
+            ]
+          },
+          "uploader": {
+            "title": "Uploader",
+            "description": "whether they are responsible for data catalog entry upload.",
+            "type": "boolean",
+            "default": false
+          }
+        }
+      }
+    },
+    "species": {
+      "title": "Species",
+      "description": "",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name_eng": {
+            "title": "Common Name (English)",
+            "description": "common name in english",
+            "type": "string"
+          },
+          "name_loc": {
+            "title": "Local Name",
+            "description": "local name at fieldsite (i.e. mono cariblanco)",
+            "type": "string"
+          },
+          "name_sci": {
+            "title": "Latin Name (Genus species)",
+            "description": "Linnean Genus species",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "locations": {
+      "title": "Locations",
+      "description": "Info about location(s) of data collection.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "country": {
+            "title": "Country",
+            "description": "",
+            "type": "string",
+            "format": "country"
+          },
+          "region": {
+            "title": "State/Province/Region",
+            "description": "if applicable",
+            "type": "string"
+          },
+          "city": {
+            "title": "City or Town",
+            "description": "nearest city if applicable",
+            "type": "string"
+          },
+          "park": {
+            "title": "Park/Protected Area",
+            "description": "name of park or protected area if applicable",
+            "type": "string"
+          },
+          "field_station": {
+            "title": "Field Station",
+            "description": "name of field station i.e. BCI or Mpala or KOB if applicable",
+            "type": "string"
+          },
+          "lat_long": {
+            "title": "Lat Long",
+            "description": "Latitude (e.g. 9.167) and Longitude (e.g. 47.679) in Decimal Degrees",
+            "type": "string",
```

Comment:
Right now, I have lat-long as a string and it's unrestricted format.

The examples have this format `7.4693 N; 81.7568 E`. I could implement some regex enforcement of that format, or just have lat and long as two separate fields.